packages/core/core/src/services/server/index.ts

```diff
@@ -17,7 +17,7 @@ const healthCheck: Core.MiddlewareHandler = async (ctx) => {
 
 const createServer = (strapi: Core.Strapi): Modules.Server.Server => {
   const app = createKoaApp({
-    proxy: strapi.config.get('server.proxy'),
```

Comment:
Shouldn't this change have a dedicated codemod?

If it's meant for another PR, could we add a TODO so that we don't forget?

Reply:
yep, already created a jira ticket but I'll mention it here too

Reply:
Hum if we just have strapi.proxy set can't we consider that it's should be the proxy applied for everything like a shorthand version ? that could be super convenient 🤔

Reply:
I don't have time to write tests for it before I'm off, but I did this as a follow-up convenience feature, and someone can hopefully finish it and update the docs: https://github.com/strapi/strapi/pull/20537

Reply:
Oh wait, I think I misunderstood. You mean the strapi.proxy could be like `string | { fetch, http, https }` and if it's a string it is treated as global?